tools/accuracy_checker/accuracy_checker/annotation_converters/fewrel.py

```diff
@@ -0,0 +1,389 @@
+"""
+Copyright (c) 2019 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from collections import namedtuple
+
+import numpy as np
+
+from ..representation import TextClassificationAnnotation, RelationClassificationAnnotation
+from ..utils import read_json
+from ..config import PathField, NumberField, BoolField
+
+from .format_converter import BaseFormatConverter, ConverterReturn
+from ._nlp_common import get_tokenizer, CLS_ID, SEP_ID
+
+
+class ERNIEFewrelConverter(BaseFormatConverter):
+    __provider__ = "ernie_fewrel"
+    annotation_types = (TextClassificationAnnotation, )
```

Comment:
```suggestion
    annotation_types = (RelationClassificationAnnotation, )
```

Reply:
I tried this, this case not work. Maybe RelationClassificationAnnotation must be registered something else, especially for metrics calculation?

Reply:
Which error does this line give?
It is strange for me, because it is just helper for LD Workbench, which converters are suitable for them and which not.
It will be ok,even if not provide it at all :)

Reply:
I need to reproduce this case. As I remember, there are no metrics calculators in case of  RelationClassificationAnnotation. I'll return later with details.

Reply:
it is diferent, metric and converter are not connected

Reply:
Yes, metric and converter are not connected, so I removed this code.